fs_test.go

```diff
@@ -61,6 +61,29 @@ func TestNewVHostPathRewriterMaliciousHost(t *testing.T) {
 	}
 }
 
+func TestInMemoryCache(t *testing.T) {
+	var ctx RequestCtx
+	var req Request
+	req.Header.SetMethod("GET")
+	req.Header.Add("Accept-Encoding", "gzip")
+	req.SetRequestURI("http://fasthttp.great/fs.go")
+	ctx.Init(&req, nil, nil)
+
+	fs := &FS{
+		Root:          "./",
+		Compress:      true,
+		InMemoryCache: true,
+	}
+	fs.NewRequestHandler()(&ctx)
+
+	if len(ctx.Response.Body()) == 0 {
+		t.Fatal("zero body for existing file")
+	}
+	if s := ctx.Response.Header.Peek("Content-Encoding"); !bytes.Equal(s, []byte("gzip")) {
+		t.Fatal("Uncompressed")
+	}
```

Comment:
Maybe check the actual result body as well? And check that multiple requests return the same content. And maybe check that it actually uses the in memory cache by changing the file on disk and checking that the old content is returned. And test `Accept-Ranges`.

Reply:
I don't push the good test file. This was another test. I will replace it. Sorry.